src/models/user.dao.js

```diff
@@ -0,0 +1,31 @@
+const { appDataSource } = require("./data-source");
+
+const createUser = async (email, password) => {
+  await appDataSource.query(
+    `
+    INSERT INTO users (
+      email,
+      password
+    ) VALUES (
+      ?,
+      ?
+    )
+    `,
+    [email, password]
+  );
+};
+
+const getUserByEmail = async (email) => {
+  const [user] = await appDataSource.query(
+    `
+      SELECT *
+      FROM users u
+      WHERE u.email = ?
+    `,
+    [email]
+  );
+
+  return user;
+};
```

Comment:
단순 email을 통해 유저의 정보 유무를 판단하는 쿼리라고 판단됩니다.

지금은 테이블의 컬럼이 4개여서 와일드카드를 사용했을 때의 성능 차이는 크게 없을 것입니다.

user 정보를 판단하기 위해 모든 정보를 조회하는 것이 아닌 id 컬럼으로도 충분히 user 데이터를 조회할 수 있을 것 같습니다~

```suggestion
const getUserByEmail = async (email) => {
  const [user] = await appDataSource.query(
    `
      SELECT u.id
      FROM users u
      WHERE u.email = ?
    `,
    [email]
  );

  return user;
};
```